content/concepts/basics/041-content-query.md

```diff
@@ -3,3 +3,29 @@ title: "Content Query"
 metaTitle: "sensenet - Content Query"
 metaDescription: "sensenet Content Query"
 ---
+
+One of the main features of sensenet is Content Query. Repository is a **query based system**. This means that every content is delivered by a very sophisticated and fast query engine.
+
+As a user, a developer or a repository administrator you get a simple but powerful content query language (CQL). You can use this language to find, filter and sort content even in a huge and constantly expanding repository. As a developer you get an easy-to use API as well, that is capable of serving any need in your custom code.
+
+# Indexing
+
+For every content [link] the field values can be indexed so that when you search for a term the corresponding content with the related field value will appear in the result set. It is also possible to search by explicitly defining the field whose value we want to search for. The way a specific field of a content is indexed is defined in the CTD Field Definition.
```

Comment:
```suggestion
For every content [content](/concepts/content-management) the field values can be indexed so that when you search for a term the corresponding content with the related field value will appear in the result set. It is also possible to search by explicitly defining the field whose value we want to search for. The way a specific field of a content is indexed is defined in the CTD Field Definition.
```